### ISSUE TITLE
add uprofiles

### ISSUE THREAD

**Issue by devahmadmohammadi** [s3bubble/Streamium-Netflix-Theme#87]
i would like to have the ability to add profiles similar to netflix so that they can have different profile within the user account

**Comment by realwildgirls** [comment]
Yeah I'm looking for same solution. Let me know if you found any plug-in that might work .

Terry
